GOVERNANCE.md

```diff
@@ -0,0 +1,111 @@
+# Keycloak Governance
+
+* [Vision](#vision) 
+* [Maintainers](#maintainers) 
+* [Contributing](#contributing)
+
+## Vision
+
+Keycloak aims to be easy to use and lightweight. The project was founded to make it easy for application developers 
+to secure modern applications and services.
+
+The 80/20 rule is a core part of the vision behind Keycloak. Stating that 80% of requirements come from roughtly 20%
+of the use-cases. We strongly believe if Keycloak would support all use-cases out of the box it would become bloated 
```

Comment:
use-cases is two words: use cases
out of the box is jargon, generally technical communications use the phrase:  "by default"

Reply:
Fixing